docs/windows.md

```diff
@@ -0,0 +1,60 @@
+# Experimental Windows support
+
+As from `v0.13.0-rc2`, Buildkit now has exprimental support on Windows. Both `buildctl.exe` and `buildkitd.exe` binaries are being released for testing purposes.
+
+We will apprecate any feedback by [opening an issue here](https://github.com/moby/buildkit/issues/new), as we stabilize the product, especially `buildkitd.exe`.
+
+## Quick start guide
+
+### Platform requirements
+
+- Architecture: `amd64`, `arm64` (binaries available but not officially tested yet).
+- Supported OS: Windows Server 2022, Windows 11.
+- Base images: `ServerCore:ltsc2022`, `NanoServer:ltsc2022`.
+
+### Setup Instructions
+
+**Dependency:** `containerd` v1.7.12+
+
+> **NOTE:** all these requires running as admin (elevated) on a PowerShell terminal.
+
+1. Setup `containerd` by following [the setup instructions here](https://github.com/containerd/containerd/blob/main/docs/getting-started.md#installing-containerd-on-windows). (_Currently, we only support the `containerd` worker_.)
+1. Start the `containerd` service, if not yet started.
+1. Setup `buildkit` binaries:
```

Comment:
Could we split this to two parts, download and setup. First part could be "Download and extract windows release tar.gz from https://.... . Alternatively you can use following script:"